sops/data_sops_file.go

```diff
@@ -55,6 +55,8 @@ func dataSourceFileRead(d *schema.ResourceData, meta interface{}) error {
 			format = "json"
 		case ".yaml":
```

Comment:
```suggestion
		case ".yaml", ".yml":
```
Minor change, but this would be easier to read than having multiple cases with the same logic.

Reply:
done !